### PR TITLE
[#35] Fixes Errors with Spaces in Filename/Directories

### PR DESCRIPTION
**Pull request by notdevinclark**
#35 

_this PR does the following_

* Updates the verbose function to handle spaces in filename/directories
* Updates creation to handle spaces
* Updates saving to handle spaces
* Adds tests for all changes
* Slight tweak to test name for matching consistency
* Also makes tests related to spaces more robust by making them test file name and directories with spaces

### CODE REVIEW

File: lib/mogrify.ex [hunk 1]
```diff
@@ -59,12 +59,12 @@ defmodule Mogrify do
   end
 
   defp arguments_for_saving(image, path) do
-    base_arguments = ~w(-write #{path} #{String.replace(image.path, " ", "\\ ")})
+    base_arguments = ~w(-write) ++ [path] ++ [image.path]
```
Comment (talklittle) [inline lib/mogrify.ex]: May as well get rid of the `~w()` syntax here, and just provide a single list instead of concatenating lists.

Comment (notdevinclark) [inline lib/mogrify.ex]: Sure thing. I do admit seeing that did feel weird leaving that one there like that, I'll fix that and push up a change.

**Comment by talklittle** [comment]
Thanks a lot for the PR and especially the unit tests.

Does this also handle spaces in the directory name leading up to the filename?

**Comment by notdevinclark** [comment]
@talklittle, no problem, glad I was able to help. I believe it will also work with the path name with spaces as well since it is no longer trying to do the replace with the path or filename anymore. I will do some more testing though just to be sure.

**Comment by notdevinclark** [comment]
Okay, I updated the tests that did anything with spaces to also use a directory with spaces in the name.

I made this a separate commit since it might require feedback/clean up.

Let me know if anything needs to be updated or changed.

**Comment by talklittle** [comment]
@notdevinclark Thanks a lot! I'll release a new Mogrify version soon.

**Comment by notdevinclark** [comment]
Sweet, sounds good. I'll keep an eye out for it. Thanks again.

**Pull request merged**